lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -347,357 +280,387 @@
-            font-size: 34px;
-            line-height: 42px;
-            font-weight: 300;
+      .course-info {
+        display: block;
+        @include float(left);
+        width: flex-grid(4);
+        padding: 0;
+        margin-top: ($baseline/2);
 
-            &:hover, &:focus {
-              text-decoration: none;
-            }
-          }
+        [class*="info-"] {
+          color: $gray-d1;
+          @extend %t-title6;
+          display: inline-block;
         }
 
-        .course-status {
-          background: $yellow;
-          border: 1px solid $border-color-2;
-          box-shadow: 0 1px 0 0 rgba(255,255,255, 0.6);
-          margin-top: 17px;
-          margin-right: flex-gutter();
-          padding: ($baseline/4);
-          width: flex-grid(8);
-          float: left;
-          @include box-sizing(border-box);
-
-          p {
-            color: $lighter-base-font-color;
-            font-style: italic;
-            letter-spacing: 1px;
-            text-align: center;
-          }
+        .info-date-block {
+          @extend %t-title7;
+          color: $gray-l1;
+          display: block;
         }
+      }
 
-        .course-status-completed {
-          background: $gray-l3;
-          color: $very-light-text;
+      .wrapper-course-actions {
+        display: block;
+        @include float(right);
+        width: flex-grid(8);
+        padding: 0;
+        margin-top: ($baseline/2);
+      }
 
-          p {
-            color: #222;
+      .course-actions {
+        position: relative;
+        top: 50%;
+        transform: translateY(-50%);
 
-            span {
-              font-weight: bold;
-            }
-          }
-        }
+        // UI: course item actions
 
-        .enter-course {
-          @include button(simple, $button-color);
+        .action {
           @include box-sizing(border-box);
+          @include margin-right($baseline/2);
+          display: inline-block;
+          float: right;
+          min-width: ($baseline*2);
+          color: $gray-l3;
           border-radius: 3px;
-          display: block;
-          @include float(left);
-          font: normal 15px/1.6rem $sans-serif;
-          letter-spacing: 0;
-          padding: 6px 32px 7px;
-          text-align: center;
-          margin-top: 16px;
+          padding: 12px;
+          border: 1px solid $white;
+          
+          &:hover, &:focus {
+            color: #333;
+            border: 1px solid $gray-l4;
+          }
 
-          &.archived {
-            @include button(simple, $button-archive-color);
-            font: normal 15px/1.6rem $sans-serif;
-            padding: 6px 32px 7px;
+          // STATE: is-disabled
+          &.is-disabled {
+            color: #808080;
+            color: $gray-l4;
+          }
 
-            &:hover, &:focus {
-              text-decoration: none;
-            }
+          // TYPE: facebook share
+          &.action-facebook, {
+            color: #3b5998;
```

Comment:
maybe we want a variable for $facebook-blue if we are going to use it elsewhere to keep it consistent?